src/common/components/game/TutorialIntroScreen.tsx

```diff
@@ -0,0 +1,61 @@
+import { Button, Dialog, DialogActions, DialogContent, DialogTitle } from '@material-ui/core';
+import { withStyles, WithStyles } from '@material-ui/core/styles';
+import { CSSProperties } from '@material-ui/core/styles/withStyles';
+import * as React from 'react';
+
+import { TUTORIAL_INTRO_Z_INDEX } from '../../../common/constants';
+
+interface TutorialIntroScreenProps {
+  onClickEndGame: () => void;
+}
+
+interface TutorialIntroScreenState {
+  open: boolean;
+}
+
+class TutorialIntroScreen extends React.Component<TutorialIntroScreenProps & WithStyles, TutorialIntroScreenState> {
+  public static styles: Record<string, CSSProperties> = {
+    modal: {
+      zIndex: TUTORIAL_INTRO_Z_INDEX
+    }
+  };
+
+  public state = {
+    open: true
+  };
+
+  public render(): JSX.Element {
+    return (
+      <Dialog
+        open={this.state.open}
+        className={this.props.classes.modal}
+      >
+        <DialogTitle>Wordbots Gameplay Tutorial</DialogTitle>
+        <DialogContent>
+          <p>Welcome to the Wordbots Tutorial!</p>
+          <p>This tutorial should take just 3 to 5 minutes to complete and will teach you all the basic gameplay mechanics required to play the game of Wordbots, including how to place robots and structures onto the board, how to use events to tilt the game in your favor, and the basic steps that go into each turn of Wordbots.</p>
```

Comment:
nit: I'd make some small copy changes:
- "robots and structures" -> "robots"
– "into each turn of Wordbots" -> "into each turn"